docs/dev/README.md

```diff
@@ -0,0 +1,70 @@
+# Quick start guide for Assisted Service Developer
+
+Over time, assisted-service has been adapted to different scenarios, and has become one more piece of many other subsystems in which it works integrated with other Openshift components.
+
+For this reason, depending on the development we are working on, we have different scenarios for developers that allow us greater flexibility for our daily work. For example, there will be cases in which we are only interested in doing some specific tests on the API and this can be covered with a light environment, but in other cases we may need a complete Openshift cluster with its control plane and worker nodes, in which case the scenario will be more complex and heavy.
+
+The frequently used scenarios are described below.
+
+## Testing simple changes to the REST service
+[Detailed documentation](../../deploy/podman/README.md)
+
+In this scenario the goal is to run small tests with the assisted-installer REST API. This can be done in a local environment by starting the assisted-service with podman.
```

Comment:
Should we make clear the `make deploy-onprem` command just brings up the environment and show an example of what type of tests and how they can be run? i.e. querying the API

Reply:
Agreed, some examples would be nice. Additionally, a document about the various tests and queries that can be run would be great to have as a reference too (but this can be added later).